notebooks/sparse/hgnn.ipynb

```diff
@@ -87,6 +76,8 @@
         "A [**hypergraph**](https://en.wikipedia.org/wiki/Hypergraph) consists of *nodes* and *hyperedges*.  Contrary to edges in graphs, a *hyperedge* can connect arbitrary number of nodes.  For instance, the following figure shows a hypergraph with 11 nodes and 5 hyperedges drawn in different colors.\n",
         "![](https://data.dgl.ai/tutorial/img/hgnn/hypergraph4.PNG)\n",
         "\n",
+        "Hypergraphs are particularly useful when the relationships between data points within the dataset is not binary.  For instance, more than products can be co-purchased together in an e-commerce system, so the relationship of co-purchase is better described as a hypergraph rather than a normal graph.\n",
```

Comment:
"more than products can be co-purchased together in an e-commerce system ..." <--- broken sentence.

Reply:
Fixed.  Good catch.